step-controller/step-controller-backend/src/test/resources/step.properties

```diff
@@ -79,5 +79,13 @@ plugins.FunctionPackagePlugin.maven.repository.central.url=https://repo1.maven.o
 #ui.authenticator.ldap.ssl.pathToJks=src/test/resources/ldap.jks
 #ui.authenticator.ldap.ssl.jksPassword=ldap
 
-plugins.timeseries.resolution.period=1000
-plugins.timeseries.flush.period=2000
+#timeseries.resolution.period=1000

```

Comment:
I've renamed these properties. We should rename them in step-distribution and step-enterprise-distribution too